### PR TITLE
RESOURCE-121 Support azure service bus topics

### LINKED ISSUE #560: Service Bus Topics

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description
Support azure service bus topic(s)

### Issues Resolved
#560 

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,6 +234,10 @@ The following is a list of static resources.
 - [azure_security_center_policies](docs/resources/azure_security_center_policies.md)
 - [azure_sentinel_incidents_resource](docs/resources/azure_sentinel_incidents_resource.md)
 - [azure_sentinel_incidents_resources](docs/resources/azure_sentinel_incidents_resources.md)
+- [azure_service_bus_namespace](docs/resources/azure_service_bus_namespace.md)
+- [azure_service_bus_namespaces](docs/resources/azure_service_bus_namespaces.md)
```
Comment (soumyo13) [inline README.md:238]: These are not required. these are already there in the https://github.com/inspec/inspec-azure/pull/562/files PR.

**Comment by soumyo13** [comment]
Please remove the changes of the azure_service_bus_namespace resource. This is a separate resource.

File: docs/resources/azure_service_bus_topics.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_service_bus_topics` resource block returns all Azure Service Bus Topics within a project.
+
+`name`, `namespace_name` and `resource_group` is a required parameter.
```
Comment (soumyo13) [inline docs/resources/azure_service_bus_topics.md:31]:
```suggestion
`name`, `namespace_name`, and `resource_group` are the required parameters.
```

**Review by soumyo13 — CHANGES_REQUESTED** [review]

**Review by soumyo13 — APPROVED** [review]

**Review by IanMadd — APPROVED** [review]

**Comment by IanMadd** [comment]
I deleted the azure_service_bus_namespace.md and azure_service_bus_namespaces.md files. Let me know if this is wrong and I can add them back.

**Pull request merged**